app/assets/javascripts/sanctuaries.js

```diff
@@ -0,0 +1,11 @@
+$(document).on('turbolinks:load', function () {
+  activateSanctuariesChosen();
+});
+
+function activateSanctuariesChosen() {
```

Comment:
It looks like there are a lot of unstyled selects in the app. Should I make a card for updating all to use Chosen and run through the same code? There's code in app/javascripts/admin_friend_form.js that's a little more complicated that could run through this [or vice versa] already.

Reply:
@rsl let me think on this one.  I remember somewhat preferring the unstyled selects in some cases during the initial build out this summer, so I will want to go through the app and revisit them.  I have been interested in streamlining the Chosen implementation, so I will definitely mull it over, thank you!

Reply:
this one good?

Reply:
@rsl yes, definitely want to use it here.  Anything that could have more than 20 options, I have been implementing Chosen dropdowns.